src/crypto-api.ts

```diff
@@ -713,7 +713,8 @@ export interface CrossSigningKeyInfo {
  * Recovery key created by {@link CryptoApi#createRecoveryKeyFromPassphrase}
```

Comment:
not a new issue, but this documentation is incomplete. This interface is also the return type of `CreateSecretStorageOpts.createSecretStorageKey`.